master/security/namespace-policy.md

```diff
@@ -1,15 +1,15 @@
 ---
-title: Use namespace in policy rules
+title: Use namespace in policy
 description: Use namespaces and namespaceSelectors in Calico network policy to group or separate resources. Use policy rules to allow or deny traffic to/from pods belonging to specific namespaces.
```

Comment:
description: Use namespaces and namespaceSelectors in Calico network policy to group or separate resources. Use policy to allow or deny traffic to/from pods in specific namespaces.

Reply:
Fixed